tests/docker/nodejs/BUILD

```diff
@@ -14,6 +14,7 @@
 
 load("//contrib:test.bzl", "container_test")
 load("//nodejs:image.bzl", "nodejs_image")
+load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_binary")
```

Comment:
run buildifier to order these loads properly

Reply:
Done.